### ISSUE TITLE
Kiwix Icon not really visible with Firefox Dark-mode

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-js#881]
See my screenshot
![image](https://user-images.githubusercontent.com/1029718/183257022-8bd42b2e-e205-4adc-afb6-1a17d2f33db4.png)

Probably a white background should be forced.

**Comment by Jaifroid** [comment]
This version with a white silhouette around it works almost everywhere. On a completely white background it could look a little strange, but I don't know anyone who has a completely white taskbar...
<img width="400" alt="image" src="https://user-images.githubusercontent.com/4304337/183310997-76581585-c387-490b-bc79-d99a07dc29b0.png">

**Comment by blakecooper** [comment]
Hi, I'd like to start contributing to this project. If this is still needed, could I be assigned to this? Thanks!

**Comment by mossroy** [comment]
Sure, thanks.
Please read and follow the contributing guidelines in CONTRIBUTING.md file.

**Comment by Jaifroid** [comment]
@blakecooper The icon illustrated in my comment above is this one: https://github.com/kiwix/kiwix-js-windows/blob/master/www/img/icons/kiwix-64.png and https://github.com/kiwix/kiwix-js-windows/blob/master/www/img/icons/kiwix-64.ico .

**Comment by blakecooper** [comment]
I just submitted a pull request for this issue: https://github.com/kiwix/kiwix-js/pull/884
